posts/inside-rust/2024-09-04-electing-new-project-directors.md

```diff
@@ -0,0 +1,84 @@
+
+We want to make sure the Project Directors can effectively represent the project as a whole, so we are soliciting input from the whole project.
+The elections process will go through two phases: Candidate Gathering and Election.
+Read on for more detail about how these work.
+
+[Role Description]: https://github.com/rust-lang/leadership-council/blob/main/roles/rust-foundation-project-director.md
+
+## Candidate Gathering
+
+The first phase is beginning right now.
+In this phase, we are inviting the members of all of the top level Rust teams and their subteams to nominate people who will make good project directors.
+The goal is to bubble these up to the Council through each of the top-level teams.
+You should be hearing from your Council Representative soon with more details, but if not, feel free to reach out to them directly.
+
+Each team is encouraged to suggest candidates.
+Since we are electing two new directors, it would be ideal for teams to nominate at least two candidates.
+Nominees can be anyone in the project and do not have to be a member of the team who nominates them.
+
+The candidate gathering process will be open until October 9, at which point each team's Council Representative will share their team's nominations and reasoning with the whole Leadership Council.
+At this point, the Council will confirm with each of the nominees that they are willing to accept the nomination and fill the role of Project Director.
+Then the Council will publish the set of candidates.
+
+This then starts a ten day period where members of the Rust Project are invited to share feedback on the nominees with the Council.
+This feedback can include reasons why a nominee would make a good project director, or concerns the Council should be aware of.
+
+The Council will announce the set of nominees by October 11 and the ten day feedback period will last until October 21.
+Once this time has passed, we will move on to the election phase.
+
+## Election
+
+The Council will meet during the week of October 21 to complete the election process.
+In this meeting we will discuss each candidate and once we have done this the facilitator will propose a set of two of them to be the new Project Directors.
+The facilitator puts this to a vote, and if the Council unanimously agrees with the proposed pair of candidates then the process is completed.
+Otherwise, we will give another opportunity for council members to express their objections and we will continue with another proposal.
+This process repeats until we find two nominees who the Council can unanimously consent to.
+The Council will then confirm these nominees through an official vote.
+
+Once this is done, we will announce the new Project Directors.
+In addition, we will contact each of the nominees, including those who were not elected, to tell them a little bit more about what we saw as their strengths and opportunities for growth to help them serve better in similar roles in the future.
```

Comment:
```suggestion
In addition, we will contact each of the nominees, including those who were not elected, to offer feedback.
For those who accept feedback we will follow up to tell them a little bit more about what we saw as their strengths and opportunities for growth to help them serve better in similar roles in the future.
```

Last time around a few nominees specifically did not want feedback. This time around I think we should make it opt-in.